src/Carbon/Traits/Difference.php

```diff
@@ -391,6 +391,14 @@ public function diffForHumans($other = null, $syntax = null, $short = false, $pa
      *
      * Get the difference in a human readable format in the current locale from current instance to an other
      * instance given (or now if null given).
+     *
+     * @param null $other
+     * @param null $syntax
+     * @param bool $short
+     * @param int  $parts
+     * @param null $options
```

Comment:
There are no null-types, it would mean you can only pass null as those parameters, please copy types and descriptions from `diffForHumans`. Thanks. https://github.com/briannesbitt/Carbon/pull/1455/files#diff-39a84c4c7e44c741019124ce379a98b5R352